docs/codeql/codeql-language-guides/basic-query-for-java-code.rst

```diff
@@ -1,18 +1,33 @@
 .. _basic-query-for-java-code:
 
-Basic query for Java code
-=========================
+Basic query for Java and Kotlin code
```

Comment:
Ideally we'd also update this article name in the "CodeQL for Java and Kotlin" article, but that will be easier to do once one of these PRs is merged.